deps/publishing/src/logseq/publishing/html.cljs

```diff
@@ -121,6 +121,7 @@ necessary db filtering"
         }
       }(window.location))"]
             ;; TODO: should make this configurable
```

Comment:
Missed this last review but app should load first

Reply:
Thanks, is there anything you need me to do @logseq-cldwalker ?

Reply:
nope but thanks for asking